core/imageroot/var/lib/nethserver/node/install-finalize.sh

```diff
@@ -155,26 +157,23 @@ A. To join this node to an already existing cluster run:
 
    For instance:
 
-      join-cluster https://cluster.example.com eyJhbGc...NiIsInR5c
+      join-cluster https://leader.example.com eyJhbGc...NiIsInR5c
 
 B. To initialize this node as a cluster leader run:
 
       create-cluster <vpn_endpoint_address>:<vpn_endpoint_port> [vpn_cidr] [admin_password]
 
    For instance:
 
-      create-cluster $(hostname -f):55820 10.5.4.0/24 ${ADMIN_PASSWORD}
+      create-cluster leader.example.com:55820 10.5.4.0/24 Nethesis,1234
```

Comment:
Use the  ${ADMIN_PASSWORD} because it can be changed from the install phase.

I also prefer to see here the hostname and not a fixed value.